pandas/core/internals/construction.py

```diff
@@ -538,7 +538,8 @@ def _list_of_series_to_arrays(data, columns, coerce_float=False, dtype=None):
 def _list_of_dict_to_arrays(data, columns, coerce_float=False, dtype=None):
     if columns is None:
         gen = (list(x.keys()) for x in data)
-        sort = not any(isinstance(d, OrderedDict) for d in data)
+        additional = dict if PY36 else OrderedDict
```

Comment:
maybe do `(dict, OrderedDict) if PY36 else OrderedDict` here and use that as the class to check on the line below?

Reply:
done